lib/main.dart

```diff
@@ -1,69 +1,76 @@
-import 'dart:ui';
-
 import 'package:flutter/material.dart';
+import 'package:sampleproject/secondscreen.dart';
+import 'package:sampleproject/thirdClass.dart';
+import 'package:sampleproject/fourthscreen.dart';
+
+extension ColorExtension on String {
```

Comment:
Please move this method into another separate file, so that we can use this method in other places as well